content/blog/how-to-backup-and-restore-stateful-applications-on-kubernetes-using-kasten-k10-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,660 @@
+
+How to access the K10 Dashboard:
+
+To establish a connection to it use the following `kubectl` command:
+
+`kubectl --namespace kasten-io port-forward service/gateway 8080:8000`
+
+The Kasten dashboard will be available at: `http://127.0.0.1:8080/k10/#/`
+```
+
+With above commands, Kasten K10 is installed to the namespace *'kasten-io'* in the cluster. To validate the installation, typing the following command to watch the status of all Pods. Helm installs a list of Pods to the namespace. It takes a while before all those Pods start running.  
+
+```shell
+$ kubectl  get pods -n kasten-io -w
+NAME                                    READY   STATUS    RESTARTS   AGE
+aggregatedapis-svc-6fc8fcf7bd-cdw8p     1/1     Running   0          15m
+auth-svc-6fcb76d7df-pt748               1/1     Running   0          15m
+catalog-svc-7c6f8b76fb-bsdqn            2/2     Running   0          15m
+controllermanager-svc-5fffc97d7-b5whv   1/1     Running   0          15m
+crypto-svc-8568584f9f-br8kn             4/4     Running   0          15m
+dashboardbff-svc-b58b6d8cd-gnt5n        2/2     Running   0          15m
+executor-svc-cb5fd4698-7zqjg            1/1     Running   0          15m
+executor-svc-cb5fd4698-n27d5            1/1     Running   0          15m
+executor-svc-cb5fd4698-rvj4v            1/1     Running   0          15m
+frontend-svc-6c5677595b-9tsmj           1/1     Running   0          15m
+gateway-54d778c955-n9wt5                1/1     Running   0          15m
+jobs-svc-668b76cb86-q27nk               1/1     Running   0          15m
+k10-grafana-889ff545b-g7px7             1/1     Running   0          15m
+kanister-svc-76cdb967bd-hkhql           1/1     Running   0          15m
+logging-svc-79599589f6-hdsp5            1/1     Running   0          15m
+metering-svc-55f84f7766-rsm5f           1/1     Running   0          15m
+prometheus-server-689ccf5f57-j9hpz      2/2     Running   0          15m
+state-svc-b4b996d9b-jnbrl               3/3     Running   0          15m
+```
+
+A﻿fter all the Pods are in running states, edit the service *gateway* to change its service type from *ClusterIP* to *NodePort*. This will generate a service port and expose the service via the configured gatway host name plus the generated port.
+
+```shell
+$ kubectl edit svc gateway -n kasten-io
+…
+spec:
+  selector:
+    service: gateway
+  sessionAffinity: None
+  type: NodePort
+…
+service/gateway edited
+```
+
+T﻿yping the following command to get the *gateway* service endpoint:
+
+```shell
+$ kubectl get svc gateway -n kasten-io -o jsonpath={.metadata.annotations.hpecp-internal-gateway/8000}
+gl-tor-upc-cp-gw-node1.customer.yyz.gl-hpe.local:10021
+```
+
+T﻿he Kasten K10 service dashboard can then be accessed by pointing your browser to the URL '*http://gl-tor-upc-cp-gw-node1.customer.yyz.gl-hpe.local:10021/k10/#/'* :
+
+![](/img/k10-login.png)
+
+C﻿lick *Accept Terms* after specifying your email and company name, you will be landed to Kasten K10 Dashboard:
```

Comment:
Change to:
Click Accept Terms after specifying your email and company name. This will land you on the Kasten K10 dashboard."